test-data/unit/check-multiple-inheritance.test

```diff
@@ -461,6 +461,128 @@ class Combo(Base2, Base1): ...
 [out]
 main:10: error: Definition of "NestedVar" in base class "Base2" is incompatible with definition in base class "Base1"
 
+[case testMultipleInheritance_NestedVariableOverriddenWithCompatibleType]
+
```

Comment:
No need for empty line after test case header (here and everywhere below).

Reply:
👍